tests/unit/dataactcore/test_populate_subaward_table.py

```diff
@@ -0,0 +1,635 @@
+        'sub_high_comp_officer3_full_na': sub_contract.top_paid_fullname_3,
+        'sub_high_comp_officer3_amount': sub_contract.top_paid_amount_3,
+        'sub_high_comp_officer4_full_na': sub_contract.top_paid_fullname_4,
+        'sub_high_comp_officer4_amount': sub_contract.top_paid_amount_4,
+        'sub_high_comp_officer5_full_na': sub_contract.top_paid_fullname_5,
+        'sub_high_comp_officer5_amount': sub_contract.top_paid_amount_5,
+
+        'prime_id': contract.id,
+        'internal_id': contract.internal_id,
+        'date_submitted': contract.date_submitted.strftime('%Y-%m-%d %H:%M:%S.%f'),
+        'report_type': contract.report_type,
+        'transaction_type': contract.transaction_type,
+        'program_title': contract.program_title,
+        'contract_agency_code': contract.contract_agency_code,
+        'contract_idv_agency_code': contract.contract_idv_agency_code,
+        'grant_funding_agency_id': None,
+        'grant_funding_agency_name': None,
+        'federal_agency_name': None,
+        'treasury_symbol': contract.treasury_symbol,
+        'dunsplus4': None,
+        'recovery_model_q1': str(contract.recovery_model_q1).lower(),
+        'recovery_model_q2': str(contract.recovery_model_q2).lower(),
+        'compensation_q1': None,
+        'compensation_q2': None,
+        'high_comp_officer1_full_na': contract.top_paid_fullname_1,
+        'high_comp_officer1_amount': contract.top_paid_amount_1,
+        'high_comp_officer2_full_na': contract.top_paid_fullname_2,
+        'high_comp_officer2_amount': contract.top_paid_amount_2,
+        'high_comp_officer3_full_na': contract.top_paid_fullname_3,
+        'high_comp_officer3_amount': contract.top_paid_amount_3,
+        'high_comp_officer4_full_na': contract.top_paid_fullname_4,
+        'high_comp_officer4_amount': contract.top_paid_amount_4,
+        'high_comp_officer5_full_na': contract.top_paid_fullname_5,
+        'high_comp_officer5_amount': contract.top_paid_amount_5,
+        'sub_id': sub_contract.id,
+        'sub_parent_id': sub_contract.parent_id,
+        'sub_federal_agency_id': None,
+        'sub_federal_agency_name': None,
+        'sub_funding_agency_id': sub_contract.funding_agency_id,
+        'sub_funding_agency_name': sub_contract.funding_agency_name,
+        'sub_funding_office_id': sub_contract.funding_office_id,
+        'sub_funding_office_name': sub_contract.funding_office_name,
+        'sub_naics': sub_contract.naics,
+        'sub_cfda_numbers': None,
+        'sub_dunsplus4': None,
+        'sub_recovery_subcontract_amt': sub_contract.recovery_subcontract_amt,
+        'sub_recovery_model_q1': str(sub_contract.recovery_model_q1).lower(),
+        'sub_recovery_model_q2': str(sub_contract.recovery_model_q2).lower(),
+        'sub_compensation_q1': None,
+        'sub_compensation_q2': None,
+    }
+    if debug and not (attr.items() <= sub.__dict__.items()):
```

Comment:
Was this supposed to get removed? Or are we leaving this debug in?

Reply:
After discussion, we decided to keep these in just for debugging purposes and it won't populate the logs.